src/httpclient.c

```diff
@@ -467,6 +467,7 @@ static void usage(const char *progname)
             "               sets the number of requests run at once (default: 1)\n"
             "  -c <size>    size of body chunk (in bytes; default: 10)\n"
             "  -d <delay>   request interval (in msec; default: 0)\n"
+            "  -D <draft>   Draft version to use for HTTP/3 (e.g. -D 29)\n"
```

Comment:
```suggestion
            "  -Q <version>  QUIC version to use for HTTP/3 in hexadecimal (e.g. ff000029)\n"
```

How about changing the name of the option to `Q` and use it for specifying versions other than version 1 and its drafts? There are proposals to use other version numbers and I think it would be reasonable to support them as well, using the same command line option.